src/h5web/vis-packs/core/line/LineVis.module.css

```diff
@@ -5,8 +5,3 @@
   min-height: 0;
   margin: 1rem;
 }
-
-.root canvas {
-  /* This sets the default color of the line */
-  color: var(--secondary-dark);
-}
```

Comment:
Left-over from a previous refactoring. We don't read the `color` property but the CSS custom color properties themselves.